wallet/dapps-communication.md

```diff
@@ -463,6 +705,81 @@ items:
 </tbody>
 </table>
 
+#### Generating New Accounts
+
+This method will generate a new random account upon request with a specified name defined by the user. The account shall be added to the wallet automatically.
+
+<table>
+<thead>
+<tr>
+<th>Direction</th>
+<th>Type</th>
+<th>Format</th>
+<th>Example</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td>Webpage to extension</td>
+<td><code>account.generateNewAccount</code></td>
+<td>
+
+```yaml
+type: object
+properties:
+  name:
+    type: string
+required:
+  - name
```

Comment:
The name attribute of an account is facultative:
```suggestion
```

Reply:
done